src/components/big-interactive-pages/editor.tsx

```diff
@@ -162,14 +204,18 @@ const exitTutorial = (persistenceState: Signal<PersistenceState>, sessionId: str
 			stale: true,
 			cloudSaveState: "SAVING",
 		};
-        saveGame(persistenceState, codeMirror.value!.state.doc.toString(), sessionId);
+		if(isNewSaveStrat.value)
```

Comment:
The desire here is to be able to enable this feature only for ppl who opt in (you, for instance, to begin with)... the code as I read it here makes me believe that this feature would result in *all* users having this activated - which would be very dangerous and no bueno!